src/lib/ankerl/svector.h

```diff
@@ -633,18 +645,11 @@ class svector {
     }
 
     void resize(size_t count) {
-        resize(count, T());
+        resize_helper(count, T());
```

Comment:
Thanks for the contribution! But I'm not sure that works as it should. This creates one `T` which is then passed to `resize_after_reserve`, and there in a loop this object will be moved multiple times

Reply:
Fair point. I will change the PR such that `T()` is directly used to initialize the new elements in this case.

Reply:
Done.